bin/feature-search.js

```diff
@@ -0,0 +1,159 @@
+#! /usr/env/bin node
+
+const fs = require( 'fs' );
+const chalk = require( 'chalk' );
+
+async function getConfigFeatures( config ) {
+	const file = `${ __dirname }/../config/${ config }.json`;
+	const json = await fs.readFileSync( file, 'utf8' );
+	const data = JSON.parse( json );
+
+	return data.features;
+}
+
+function calcLongestConfig( result ) {
+	const configLengths = result.map( ( item ) => ( item.config ? item.config.length : 0 ) );
+	return Math.max( ...configLengths );
+}
+
+function getPadding( count ) {
+	let padding = '';
+	for ( let i = 0; i < count; i++ ) {
+		padding += ' ';
+	}
+
+	return padding;
+}
+
+function sortResult( result ) {
+	result.sort( ( a, b ) => {
+		if ( a.config < b.config ) {
+			return -1;
+		}
+
+		if ( a.config > b.config ) {
+			return 1;
+		}
+
+		return 0;
+	} );
+}
+
+function outputResults( results ) {
+	const resultKeys = Object.keys( results ).sort();
+
+	if ( resultKeys.length === 0 ) {
+		console.log( 'No matching features found.' );
+		return;
+	}
+
+	for ( const key of resultKeys ) {
+		const result = results[ key ];
+		const maxLength = calcLongestConfig( result );
+
+		sortResult( result );
+
+		console.log( chalk.white.bgBlue.bold( key ) );
+		for ( const item of result ) {
+			const { config, set } = item;
+			const configStr = chalk.cyan( `\t${ config }:` );
+
+			let setStr = '';
+			if ( set === true ) {
+				setStr = chalk.green( set );
+			} else if ( set === false ) {
+				setStr = chalk.red( set );
+			} else {
+				setStr = chalk.yellow( set );
+			}
```

Comment:
Thought: we can still handle `null` in a clear way here.
```suggestion
			} else if ( set === null ) {
				setStr = chalk.yellow( '(not set)' );
			} else {
				setStr = chalk.yellow( set );
			}
```

Also, as a more general note, it may eventually be worth pushing this logic into a helper function with early returns. Something like the following could work, but might be overkill for now:
```js
function getFormattedFeatureString( set ) {
	if ( set === true ) {
		return chalk.green( set );
	}

	if (set === false ) {
		return chalk.red( set );
	}

	if ( set === null ) {
		return chalk.yellow( '(not set)' );
	}

	return chalk.yellow( set );
}
```